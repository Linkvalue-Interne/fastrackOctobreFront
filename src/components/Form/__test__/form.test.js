import React from 'react';
import { createSerializer } from 'jest-emotion';
import Adapter from 'enzyme-adapter-react-16';
import { shallow, configure } from 'enzyme';
import * as emotion from 'emotion';

import {
  Button,
  Input,
  InputFile,
  Select,
  Label,
} from '../../commons/Form/FormStyle';

expect.addSnapshotSerializer(createSerializer(emotion));

configure({ adapter: new Adapter() });

describe('formTest', () => {
  it('Button should renders with correct styles', () => {
    const tree = shallow(<Button />);
    expect(tree).toMatchSnapshot();
  });

  it('Input should renders with correct styles', () => {
    const tree = shallow(<Input />);
    expect(tree).toMatchSnapshot();
  });

  it('InputFile should renders with correct styles', () => {
    const tree = shallow(<InputFile />);
    expect(tree).toMatchSnapshot();
  });

  it('Select renders with correct styles', () => {
    const tree = shallow(<Select />);
    expect(tree).toMatchSnapshot();
  });

  it('Label renders with correct styles', () => {
    const tree = shallow(<Label />);
    expect(tree).toMatchSnapshot();
  });
});
