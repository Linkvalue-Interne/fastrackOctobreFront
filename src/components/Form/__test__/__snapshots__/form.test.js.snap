// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`formTest Button should renders with correct styles 1`] = `ShallowWrapper {}`;

exports[`formTest Input should renders with correct styles 1`] = `ShallowWrapper {}`;

exports[`formTest InputFile should renders with correct styles 1`] = `ShallowWrapper {}`;

exports[`formTest Label renders with correct styles 1`] = `ShallowWrapper {}`;

exports[`formTest Select renders with correct styles 1`] = `ShallowWrapper {}`;
